src/validatePrTitle.js

```diff
@@ -133,4 +134,6 @@ module.exports = async function validatePrTitle(
       );
     }
   }
+
+  core.setOutput('error_message', errorMessage);
```

Comment:
Do you have some reasoning on why you picked `error_message`? Generally we use camelCase, so in doubt I'd go for that here too. It would also be good to do some research if there is some existing convention from other actions that we could align ourselves to.

Reply:
Well, I only based my choice on the Conventional Changelog Action, which uses snake_case for the outputs.

https://github.com/TriPSs/conventional-changelog-action

I can look into some more famous examples, but the final decision is yours xD what do you prefer? camelCase?

Reply:
I recommend using as it is in docs, upper case and underscore

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

Reply:
Good idea, base it on docs seems like the best option imo

Reply:
Thanks for the suggestion @derberg!

I just checked the docs and it seems like they're a bit inconsistent with the casing:
1. Upper case snake case:
    - https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions
2. Lower case snake case:
    - https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-setting-an-output-parameter
    - https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-toggling-command-echoing
    - https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs (no snake case since it's a single word, but lowercase)
    - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-output-as-url
3. Kebab case:
   - https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-declaring-outputs-for-composite-actions

It seems like there's unfortunately not really a standard here. However lower case snake case seems to be the most popular one and also what convitional-changelog-action is using, so I'd vote to go for that.